### ISSUE TITLE
pg_dump: aborting because of server version mismatch (RM #4003)

### ISSUE THREAD

**Issue by dpage** [postgres/pgadmin4#2369]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/4003***
*Originally created by **Ryonez Coruscare** at 2019-02-18 00:36:38 UTC.*

My environment is using the following docker images:
postgres:11

dpage/pgadmin4

On a machine using unRaid for it's OS.

When I try to backup info from my database, pgadmin throws the following error:

<pre><code class="text">
pg_dump: server version: 11.1; pg_dump version: 10.5
pg_dump: aborting because of server version mismatch
</code></pre>

There's a related (And for some reason rejected) issue here: https://redmine.postgresql.org/issues/3843

Long story short, I can't get the binaries for postgres 11.1, they are no longer offered on the download page for linux: https://www.enterprisedb.com/download-postgresql-binaries

So I'm stumped as to why the tools in pgadmin aren't up to date. What step/s should I take, or is this something that will be fixed on pgadmins side?

**Comment by dpage** [comment]
Issue closed on Redmine.
